go.mod

```diff
@@ -7,7 +7,7 @@ require (
 	github.com/coopnorge/go-logger v0.11.3
 	github.com/grpc-ecosystem/go-grpc-middleware/v2 v2.1.0
 	github.com/iancoleman/strcase v0.3.0
-	github.com/labstack/echo/v4 v4.13.0
+	github.com/labstack/echo/v4 v4.12.0
```

Comment:
This seems like a rebase-error, as I see no reason to downgrade echo

Reply:
Fixed

Reply:
No, you did not :P